it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/pubsub/PubsubResourceManager.java

```diff
@@ -85,6 +91,8 @@ public final class PubsubResourceManager implements ResourceManager {
 
   private final SchemaServiceClient schemaServiceClient;
 
+  private final MonitoringClient monitoringClient;
```

Comment:
Nit: Let's call the variable `monitoringResourceManager` so that the idea of dependent resource manager creation is clearer?

Reply:
Monitoring client is different from resource managers in the sense that there is no underlying "resource" it manages. It doesn't have create and cleanup life cycles and it is just like a wrapper around Monitoring API library. That is why it is not called Monitoring Resource manager.